packages/engine/tests/units/state/edit/non_existing_field/src/behaviors/test.py

```diff
@@ -0,0 +1,3 @@
+def behavior(state, context):
+    """Reading non-existing field `non_existing_field`"""
```

Comment:
Same as above, don't think this is a good description

Reply:
```suggestion
    """Reading a non-existing field to ensure the return type is `None`"""
```